src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -11,17 +11,18 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "goto_instrument_parse_options.h"
 
-#include <fstream>
-#include <iostream>
-#include <memory>
-
 #include <util/exception_utils.h>
 #include <util/exit_codes.h>
 #include <util/json.h>
+#include <util/options.h>
 #include <util/string2int.h>
 #include <util/string_utils.h>
 #include <util/version.h>
 
+#include <fstream>
+#include <iostream>
+#include <memory>
+
```

Comment:
Why do we need these includes here? Debug leftovers perhaps?

Reply:
This is just a reordering of the includes due to new clang-format rules